package 二十五丶异常处理.tryCatchFinally;
/*
*处理异常：try catch 或者 throws   这两个的区别是设计层面的区别
*        如果当前方法自己能够处理，就用try catch；如果当前方法自己不能够处理，就用throws，交给上级(方法调用处)来处理
* */
public class text {
}
